cmd/rootMain.go

```diff
@@ -59,6 +67,18 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 	}
 	defer os.RemoveAll(rundir)
 
+	socketpath := filepath.Join(rundir, "api.sock")
+
+	if rootVerifyInstallFlag {
+		srv := server.ForVerifyInstall(mountpath, socketpath)
+		if err := srv.Start(); err != nil {
+			cmdutil.ErrPrintf("Install verification failed: %v\n", err)
```

Comment:
I think we should also print something when it succeeds (on stdout). After stopping the server.